docs/tutorials/motion/plan-motion-with-arm-gripper.md

```diff
@@ -0,0 +1,561 @@
+---
+title: "Plan Motion with an Arm and a Gripper"
+linkTitle: "Plan Motion with an Arm"
+weight: 20
+type: "docs"
+description: "Use the Motion Service to move robot arms and other components."
+tags: ["arm", "gripper", "motion"]
+# SMEs: William S.
+---
+
+{{< alert title="Caution" color="caution" >}}
+Be careful when instructing robot arms to move.
+Before running any code, ensure your robotic arm has enough space and that there are no obstacles.
+Also pay attention to your surroundings, double-check your code for correctness, and make sure anyone nearby is aware and alert before issuing commands to your robot.
+{{< /alert >}}
+
+Moving individual components is a good way to start using Viam, but there is so much more you can do.
+The [Motion Service](/services/motion/) enables sophisticated movements involving one or many components of your robot.
+The Motion Service is one of the "built-in" services, which means that no initial configuration is required to start planning and executing complex motion.
+All you need is a robot with a component that can move, such as a robotic arm.
```

Comment:
Let's move that to "Configure a robot" just before "Accessing the Motion Service":